bbq/vm/test/interpreter_test.go

```diff
@@ -799,21 +799,21 @@ func BenchmarkInterpreterFTTransfer(b *testing.B) {
 	amount := interpreter.NewUnmeteredIntValueFromInt64(transferAmount)
 	receiver := interpreter.AddressValue(receiverAddress)
 
-	inter, err = parseCheckAndInterpretWithOptions(
-		b,
-		realFlowTokenTransferTransaction,
-		txLocation(),
-		ParseCheckAndInterpretOptions{
-			Config:        interConfig,
-			CheckerConfig: checkerConfig,
-		},
-	)
-	require.NoError(b, err)
-
 	b.ReportAllocs()
 	b.ResetTimer()
 
 	for i := 0; i < b.N; i++ {
+		inter, err = parseCheckAndInterpretWithOptions(
+			b,
+			realFlowTokenTransferTransaction,
+			txLocation(),
+			ParseCheckAndInterpretOptions{
+				Config:        interConfig,
+				CheckerConfig: checkerConfig,
+			},
+		)
+		require.NoError(b, err)
+
```

Comment:
Was this change, now also measuring parsing and checking, intentional?

Reply:
Yes - I updated both the interpreter-test and the vm-test to benchmark all phases (including parsing and checking), to see where we stand. At the moment there is about a ~13-15% difference: compile+VM is slower by ~13-15%, because of the compilation overhead (previously was ~28-30%). I think we should be on-par with the existing interpreter with some more low-hanging optimizations and the re-use of interpreter values.